app/src/pages/inside/stepPage/modals/makeDecisionModal/elements/testItemDetails/testItemDetails.jsx

```diff
@@ -0,0 +1,131 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import Parser from 'html-react-parser';
+import commentIcon from 'common/img/comment-inline.svg';
+import asteriskIcon from 'common/img/asterisk-inline.svg';
+import { ERROR_LOGS_SIZE } from 'pages/inside/stepPage/modals/makeDecisionModal/constants';
+import { StackTraceMessageBlock } from 'pages/inside/common/stackTraceMessageBlock';
+import { uniqueId } from 'common/utils';
+import { ItemHeader } from '../itemHeader';
+import styles from './testItemDetails.scss';
+
+const cx = classNames.bind(styles);
+
+export const TestItemDetails = ({
+  item,
+  mode,
+  selectItem,
+  logs,
+  eventsInfo,
+  suggestRs,
+  hideLabels,
+  onClickLinkEvent,
+  isSelected,
+  showErrorLogs,
+}) => {
+  const [showDetails, setShowDetails] = useState(showErrorLogs);
+
+  useEffect(() => {
+    setShowDetails(showErrorLogs);
+  }, [showErrorLogs]);
+
+  const toggleDetails = () => {
+    setShowDetails(!showDetails);
+  };
+
+  return (
+    <div className={cx('test-item-details')}>
+      <ItemHeader
+        item={item}
+        mode={mode}
+        selectItem={selectItem}
+        isSelected={isSelected}
+        hideLabels={hideLabels}
+        onClickLinkEvent={onClickLinkEvent}
+        toggleDetails={toggleDetails}
+      />
+      {showDetails && (
+        <div className={cx('test-item-details-content')}>
+          {item.issue && item.issue.comment && (
+            <div className={cx('defect-comment')}>
+              <div>{Parser(commentIcon)}</div>
+              <div className={cx('defect-comment-content')}>
+                <p className={cx('defect-comment-title')}>Defect Comment</p>
+                <p className={cx('defect-comment-text')}>{item.issue.comment}</p>
+              </div>
+            </div>
+          )}
+          {showDetails &&
+            logs.slice(0, ERROR_LOGS_SIZE).map((log) => (
+              <div key={uniqueId()} className={cx('error-log')}>
+                <StackTraceMessageBlock
+                  level={log.level}
+                  designMode="dark"
+                  maxHeight={90}
+                  eventsInfo={eventsInfo}
+                >
+                  <div className={cx('similar-log')}>
+                    {suggestRs.relevantLogId === log.id && (
+                      <p className={cx('similar-log-title')}>{Parser(asteriskIcon)} Similar Log</p>
```

Comment:
Please add corresponding translations.
Rename classes to highlighted log, and provide this message from props.